t/31-client_file.t

```diff
@@ -169,7 +169,7 @@ subtest 'verify_chunks' => sub {
 
     $pieces->first->content('42')->write_content($copied_file);    #Let's simulate a writing error
     is(
-        OpenQA::Files->verify_chunks($t_dir => $copied_file),
+        OpenQA::Files->verify_chunks($t_dir => $copied_file)->message(),
         'Can\'t verify written data from chunk',
```

Comment:
Exceptions should be checked with a regex comparing the beginning, not an equality check.

Reply:
Side note: This is a weird API, `Mojo::Exception` objects are meant to be thrown, not returned.